index.html

```diff
@@ -2326,123 +2326,87 @@ <h2>Services</h2>
+where each service is described by a <a data-cite="INFRA#ordered-map">map</a>.
+Each <a>service</a> <a data-cite="INFRA#ordered-map">map</a> MUST contain
+<code><a>id</a></code>, <code>type</code>, and
+<code><a>serviceEndpoint</a></code> properties. Each service extension MAY
+include additional properties and MAY further restrict the properties associated
 with the extension.
           </p>
-        </dd>
+          <dl>
+            <dt>id</dt>
+            <dd>
+The value of the <code>id</code> property MUST be a <a>URI</a> conforming to
+[[RFC3986]]. A <a>conforming producer</a> MUST NOT produce
+multiple <code>service</code> entries with the same <code>id</code>.
+A <a>conforming consumer</a> MUST produce an error if it detects
+multiple <code>service</code> entries with the same <code>id</code>.
+            </dd>
+          <dt>type</dt>
+          <dd>
+The value of the <code>type</code> property MUST be a <a
+data-cite="INFRA#string">string</a> or an <a
+data-cite="INFRA#ordered-set">ordered set</a> of <a
+data-cite="INFRA#string">strings</a>.  In order to maximize interoperability,
+the <a>verification method</a> type and  its associated properties SHOULD be
+registered in the DID Specification Registries [[?DID-SPEC-REGISTRIES]].
+          </dd>
+          <dt><dfn>serviceEndpoint</dfn></dt>
+          <dd>
+The value of the <code>serviceEndpoint</code> property MUST be a <a
+data-cite="INFRA#string">string</a>, a <a data-cite="INFRA#string">map</a>, or
+an <a data-cite="INFRA#ordered-set">ordered set</a> composed of one or more <a
+data-cite="INFRA#string">strings</a> and/or <a
+data-cite="INFRA#string">maps</a>. All <a data-cite="INFRA#string">string</a>
+values MUST be valid <a>URIs</a> conforming to [[RFC3986]] and normalized
+according to the <a data-cite="RFC3986#section-6">Normalization and Comparison
+rules in RFC3986</a> and to any normalization rules in its applicable <a>URI</a>
+scheme specification.
+          </dd>
+        </dl>
       </dl>
 
       <p>
-It is expected that the <a>service</a> protocol is published in an open
-standard specification.
-      </p>
-
-      <p>
-For more information about security considerations regarding authentication
-<a>services</a> see Sections <a href="#method-schemes"></a>
-and <a href="#authentication"></a>.
+For more information regarding privacy and security considerations related
+to <a>services</a> see <a href="#service-privacy"></a>, <a
+href="#keep-personal-data-private"></a>, <a
+href="#did-document-correlation-risks"></a>, and <a
+href="#authentication-service-endpoints"></a>.
       </p>
 
-      <pre class="example nohighlight" title="Various services">
+      <pre class="example nohighlight" title="Usage of the service property">
 {
   "service": [{
     "id":"did:example:123#linked-domain",
     "type": "LinkedDomains", <span class="comment">// external (property value)</span>
     "serviceEndpoint": "https://bar.example.com"
-  }, {
```

Comment:
I believe I understand why you removed the extra examples -- they really only vary by "type" -- but you now have an array with only one item. Maybe add one other item back in so we have two items in the example?

Reply:
I'd love to add another example -- but to my knowledge, we don't have any other examples we can provide that have specifications attached to them and that are registered in the DID Spec Registries <-- the WG wants to see that happen for any example we put in the specification. Do you know of any service specifications that are missing from the DID Spec registries that we could add and use?